pageserver/src/tenant/storage_layer/layer_desc.rs

```diff
@@ -87,7 +86,8 @@ impl PersistentLayerDesc {
             key_range,
             lsn_range: Self::image_layer_lsn_range(lsn),
             is_delta: false,
-            is_incremental,
+            // currently all image layers are non-incremental
+            is_incremental: false,
```

Comment:
Why not remove this `is_incremental` member as well?

Reply:
hmm as in it's the same as `is_delta`... Yes, I guess I could do that.

Reply:
Done in [97d1443](https://github.com/neondatabase/neon/pull/5061/commits/97d1443ed989409456fdf410f2ada1c5987457f4)